scipy/cluster/meson.build

```diff
@@ -14,8 +14,10 @@ _hierarchy_c = custom_target('_hierarchy_c',
     #cython_blas_pxd, cython_lapack_pxd
   ],
   command: [cython, '-3', '--fast-fail',
-            '@INPUT0@', '-o', '@OUTPUT@', '-I', '@BUILD_ROOT@'],
+    '@INPUT0@', '-o', '@OUTPUT@', '-I', '@BUILD_ROOT@'
```

Comment:
This one doesn't quite look right to me. I'd either leave it unchanged or write it as:
```python
  command: [
    cython, '-3', '--fast-fail',
    '@INPUT0@', '-o', '@OUTPUT@', '-I', '@BUILD_ROOT@'
  ]
```
But that's maybe a little too verbose? Since it's a single command that was just too long to fit on a single line, and now it would take four lines.

Ah, not entirely sure, maybe your change is fine. Let me wake up and look at it again in a bit.

Reply:
I think I prefer the 4-liner above. And maybe it fits on 3 lines now? Is this too long still?
```
cython, '-3', '--fast-fail',  '@INPUT0@', '-o', '@OUTPUT@', '-I', '@BUILD_ROOT@'
```

Reply:
4-lines lgtm